frontend/src/hooks/useMdComponents.tsx

```diff
@@ -84,7 +84,7 @@ export const useMdComponents = ({
       a: ({ node, ...props }) => {
         const { href } = props
         const isExternal =
-          typeof href === 'string' && !href.startsWith(window.location.origin)
+          (href && !href.startsWith(window.location.origin)) || false
```

Comment:
Just to double check: this is type safe because `href` is defined as `string` in `props` rihgt?

Reply:
Seems it is from react-markdown Components type. href is guaranteed to be `string` according to their docs https://github.com/remarkjs/react-markdown#appendix-b-components

Reply:
Btw to follow up, the typing is defined here in the `react-markdown` package:
<img width="1459" alt="Screenshot 2023-07-25 at 5 19 39 PM" src="https://github.com/opengovsg/FormSG/assets/37061143/c4d03845-f68d-4ba2-8365-10e206d41d8a">
So href is `string | undefined`! Since other components don't have the `href` prop.

Reply:
Oh yea, what you said! (ps just saw your comment @kenjin-work)